### PR TITLE
Update nav walker to 4.3.0: Accessibility, schema and security updates

### PR DESCRIPTION
**Pull request by ronilaukkarinen**
The [nav walker](https://github.com/wp-bootstrap/wp-bootstrap-navwalker/blob/c856e941c4209581f8b90a31cc33350c758b3cfb/class-wp-bootstrap-navwalker.php) that ours is originally based on has been continuously developed further during all these years where ours got stuck. This PR will add to nav walker

- Structure and updates from 4.3.0 (without icon fonts)
- Schemas
- Aria attributes like aria-labelledby and aria-current
- Security updates for targets

This will also fix the issue #93.

I have tested out the branch and it seems to work fine. Report back if something is not working as intended.

### CODE REVIEW

File: inc/includes/nav-walker.php [hunk 1]
```diff
@@ -32,174 +83,519 @@ public function start_lvl( &$output, $depth = 0, $args = array() ) {
       $icon = ob_get_clean();
       $output .= '<button class="dropdown-toggle" aria-expanded="false" aria-label="' . get_default_localization( 'Open child menu' ) . '">';
       $output .= $icon . '</button>';
-      $output .= "\n$indent<ul class=\"sub-menu\">\n";
-    } else {
-      $output .= "\n$indent<ul>\n";
```
Comment (Niq1982) [inline inc/includes/nav-walker.php:37]: Why is this removed? There's still need for non-dropdown menus for example in footers.

Comment (ronilaukkarinen) [reply inc/includes/nav-walker.php:37]: If you look a bit further at line 103 you see it's not removed. It's replaced with new intend and labelledby.

Comment (Niq1982) [reply inc/includes/nav-walker.php:37]: Ah, my bad, the diff was confusing.
